src/syft/core/common/module.py

```diff
@@ -0,0 +1,84 @@
+# stdlib
+from collections import OrderedDict
+from typing import Any
+from typing import List
+from typing import Optional
+from typing import Union
+
+
+# circular imports when using the syft.lib.full_name_with_qualname version
+def full_name_with_qualname(klass: type) -> str:
+    return f"{klass.__module__}.{klass.__qualname__}"
+
+
+class Module:
```

Comment:
Should we put this in the torch folder if it's torch specific?

Reply:
Yes open to suggestions. I have created a seperate ticket: https://github.com/OpenMined/PySyft/issues/4747